tgui/packages/tgui/interfaces/AresInterface.js

```diff
@@ -0,0 +1,679 @@
+import { useBackend } from '../backend';
+import { Flex, Box, Section, Button } from '../components';
+import { Window } from '../layouts';
+
+const PAGES = {
+  'login': () => Login,
+  'main': () => MainMenu,
+  'announcements': () => AnnouncementLogs,
+  'bioscans': () => BioscanLogs,
+  'bombardments': () => BombardmentLogs,
+  'apollo': () => ApolloLog,
+  'access_log': () => AccessLogs,
+  'delete_log': () => DeletionLogs,
+};
+
+export const AresInterface = (props, context) => {
+  const { data } = useBackend(context);
+  const { current_menu } = data;
+  const PageComponent = PAGES[current_menu]();
+
+  return (
+    <Window theme="weyland" width={780} height={725}>
+      <Window.Content scrollable>
+        <PageComponent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+const Login = (props, context) => {
+  const { act } = useBackend(context);
+
+  return (
+    <Flex
+      direction="column"
+      justify="center"
+      align="center"
+      height="100%"
+      color="darkgrey"
+      fontSize="2rem"
+      mt="-3rem"
+      bold>
+      <Box fontFamily="monospace">ARES v3.2 Interface</Box>
+      <Box mb="2rem" fontFamily="monospace">
+        WY-DOS Executive
+      </Box>
+      <Box fontFamily="monospace">Version 8.1.1</Box>
+      <Box fontFamily="monospace">Copyright © 2182, Weyland Yutani Corp.</Box>
+
+      <Button
+        content="Login"
+        icon="user-shield"
+        width="30vw"
+        textAlign="center"
+        fontSize="1.5rem"
+        p="1rem"
+        mt="5rem"
+        onClick={() => act('login')}
+      />
+    </Flex>
+  );
+};
+
+const MainMenu = (props, context) => {
+  const { data, act } = useBackend(context);
+  const { logged_in, access_text, last_page, current_menu, access_level } =
+    data;
+
+  return (
+    <>
+      <Section>
+        <Flex align="center">
+          <Button
+            icon="arrow-left"
+            px="2rem"
+            textAlign="center"
+            mr="1rem"
+            tooltip="Go back"
+            onClick={() => act('go_back')}
+            disabled={last_page === current_menu}
+          />
+
+          <h3>
+            {logged_in}, {access_text}
+          </h3>
+
+          <Button.Confirm
+            content="Logout"
+            icon="circle-user"
+            ml="auto"
+            px="2rem"
+            bold
+            onClick={() => act('logout')}
+          />
+        </Flex>
+      </Section>
+
+      <Section>
+        <h1 align="center">Navigation Menu</h1>
+      </Section>
```

Comment:
I think you should wrap all of the sections below this section and use `<Section title="Navigation Menu">` instead.
All your subsections can be fitted so that it looks fine inline, but this'll provide a more aesthetically pleasing look